### DISCUSSION TITLE
Example for offset_surface function on python

### DISCUSSION THREAD

**Discussion by FaezAlkadi** [libigl/libigl#1835] [Q&A]
I would appreciate it if I can see an example for using [offset_surface ](https://libigl.github.io/libigl-python-bindings/igl_docs/#offset_surface) on python as follow:
- read object in stl format 
- offset object with certain distance
- write new object (offsetted object) in stl as well.

long story short, I am looking for something like [this ](https://www.alecjacobson.com/weblog/?tag=offset) but with **python** instead of **matlab**

Thank you 

@jdumas 
Since you locked this thread without giving me a chance to explain. I am editing this to include my comment.  
The reason I opened only two threads "Not over and over" is because my labtop freezed on me just after I hit submit so I thought it didn't go through!
I am a bigginer with python and I am asking to learn !!!

**Comment by jdumas** [comment]
Please stop opening threads asking the same question over and over again.
